pkg/conn/conn.go

```diff
@@ -433,6 +434,53 @@ func (mgr *Mgr) listSchedulersWith(ctx context.Context, get pdHTTPRequest) ([]st
 	return nil, err
 }
 
+// GetPDScheduleConfig returns PD schedule config value associated with the key.
+// It returns nil if there is no such config item.
+func (mgr *Mgr) GetPDScheduleConfig(
+	ctx context.Context, configKey string,
```

Comment:
better replace `configKey string` by `configKeys []string`, or even better, `cfgPointer interface{}`, so that fetching N config values doesn't require performing N `pdRequest`s.